### PR TITLE
chore: delete missing docs to redirect

### PR DESCRIPTION
**Pull request by mateusz-001**
#### Changelog entry
- Added missing redirection ([spreadsheet with redirections](https://docs.google.com/spreadsheets/d/1x8zc8MHvddvXUrRBFzgNSuMIdHcdmKpreeqRc0X7tiY/edit#gid=0))
- Deleted files `/support` folder and other files for make redirections working

### CODE REVIEW

File: pages/05.api-references/02.graphql-integration-api/04.documentation/docs.md
Comment (remziatay) [inline pages/05.api-references/02.graphql-integration-api/04.documentation/docs.md:1]: This route is not in redirects

File: pages/07.support/01.rules-for-partners/docs.md
Comment (remziatay) [inline pages/07.support/01.rules-for-partners/docs.md:1]: `/support/rules-for-partners` is not redirected

File: pages/07.support/02.certification/docs.md
Comment (remziatay) [inline pages/07.support/02.certification/docs.md:1]: `/support/certification` is not redirected

File: pages/07.support/chapter.md
Comment (remziatay) [inline pages/07.support/chapter.md:1]: `/support` is not redirected

File: pages/03.fe-development/chapter.md
Comment (remziatay) [inline pages/03.fe-development/chapter.md:1]: `/fe-development` is not redirected

File: pages/01.overview/05.promo/docs.md
Comment (remziatay) [inline pages/01.overview/05.promo/docs.md:1]: `/overview/promo` is not redirected

**Comment by remziatay** [comment]
Can we delete `graphql-explorer/graphql-explorer.md_moved` file?

**Comment by remziatay** [comment]
All internal links in homepage are broken, are we supposed to fix those?

**Comment by mateusz-001** [comment]
> All internal links in homepage are broken, are we supposed to fix those?

Homepage will be completely changed, check out this [PR](https://github.com/centrahq/centra-grav-themes/pull/92)

**Comment by mateusz-001** [comment]
I redirected all pages without redirections in spreadsheet to `centra.dev/docs` <br> `/api-references/graphql-integration-api/documentation` to `centra.dev/graphql`

**Review by remziatay — APPROVED** [review]

**Review by rebzden-centra — APPROVED** [review]

**Pull request merged**